rs/tests/system_tests.bzl

```diff
@@ -33,26 +33,33 @@ def _run_system_test(ctx):
         content = """#!/bin/bash
             set -eEuo pipefail
 
-            # for every ic-os image specified, first ensure it's in remote storage, then
-            # export its download URL as an environment variable.
-            if [ -n "$ICOS_IMAGES" ]; then
-              # split the ";"-delimited list of "envvar:filepath;envvar2:filepath2;..."
+            # Resolve any RUN_SCRIPT_ variables
```

Comment:
this won't resolve any `RUN_SCRIPT_` variables, right? only the handful (2) that we deal with explicitly

Reply:
I added this to make it clear that `RUN_SCRIPT_` variables won't make it to the test driver in the same way that other variables do. I was thinking this section would take care of any others that we do make.